policies/communication_channels.md

```diff
@@ -0,0 +1,59 @@
+# Communication channels in the Astropy Project
+
+## Preface
+
+Communication in the Astropy Project is governed by the [Community
+Code of Conduct](https://www.astropy.org/code_of_conduct.html), which
+takes precedence over anything written here. This goal of this policy is
+to write down the technical details of how email lists, Slack
+channels, forums etc. are set up.
+
+While we aim to be an inclusive and welcoming community, we
+recognize that not all users want to be involved in the technical
+details of building and running a library like Astropy. Therefore, some
+communication channels that are more geared towards users and others
+to developers. Also, in a project this size, certain issues that
+arise require privacy (e.g. personal or financial information is
+legally protected in many contries). Thus, there are communication
+channels geared towards or exclusively for developers, maintainers, or
+members of certain committees.
+
+
+## Email communication geared towards project members
+
+Mailing lists are maintained as Google groups or through other list
+servers. Unless specifically restricted, anyone worldwide can
+subscribe to these lists, and previous posts are publicly
+available. List membership is required to post messages to prevent
+spam.  In order to be as open as possible, none of these lists shall
+be moderated and nor will posts by group members be otherwise
+restricted. However, the Coordination Committee can implement a "cool
+down" period of no more than 48 hours if they deem it necessary to
+maintain a respectful and welcoming atmosphere. During this period,
+posting may be moderated or otherwise restricted by the Coordination
+Committee. The beginning and end of such a period has to be announced
+by the Coordination Committee on the affected mailing list. The
+Coordination Committee shall not implement a "cool down" within a week
+of any scheduled Astropy vote unless in the most obvious cases of Code
+of Conduct violations.
+
+
+The following email lists / aliases are restricted in memebership and
+their content is not public:
+
+- astropy-core-maintainer: Membership: Anyone with commit rights to the github/astropy/astropy repository
+- astropy-gsoc-mentors-XXX: Membership: Mentors of Astropy GSOC projects and astropy GSOC coordinator, coordination committee
+- coordinators@astropy.org: Coordination Committee
+- confidential@astropy.org: Ombudsperson
+- finance@astropy.org: Finance Committee
+
+
+## Astropy Slack channels
+Anyone can join the Astropy Slack workspace and join any number of channels, unless specifically restricted. The following channels are restricted:
+
+- gsoc-mentors: Same restriction as analogous mailing list
+
+Posting to all channels is open to all workspace members and shall
+not be restricted. The Coordination Committee can implement a "cool
```

Comment:
This is in contradiction with the existence of private channels, so maybe saying 'Posting to all public channels'